src/tools/cgset.c

```diff
@@ -88,7 +88,7 @@ STATIC int parse_r_flag(const char * const program_name, const char * const name
 	buf = strtok(copy, "=");
 	if (buf == NULL) {
 		err("%s: wrong parameter of option -r: %s\n", program_name, optarg);
-		ret = -1;
+		ret = EXIT_BADARGS;
```

Comment:
Good find.  I grepped the tools folder for calls to `usage()`, and that missed many invalid parameter cases like this.